json-sql

```diff
@@ -1,17 +1,29 @@
 #!/usr/bin/env python3
 
 import decimal
+import datetime
 import json
 import sqlite3
 import sys
 
 import pymysql
 import psycopg2
+import cx_Oracle
+import platform 
+import os
 
+# to avoid "TypeError: Object of type datetime is not JSON serializable"
+# per https://stackoverflow.com/a/35870294
+def datetime_handler(x):
```

Comment:
Nice catch on the datetime objects.
We can simplify the code by just using the logic in jsonable that you implemented on line 25.